Dockerfile

```diff
@@ -35,7 +35,13 @@ RUN mkdir -p $TMP_DIR \
 COPY Gemfile \
      Gemfile.lock ./
 
-RUN bundle --without test development
+ENV WEB_CONCURRENCY=2
+
+RUN if [ "$debug" = "true" ] ; then  \
+	WEB_CONCURRENCY=0 ; \
+fi
+
+RUN if [ "$debug" != "true" ] ; then bundle --without test development ; else bundle --without test ; fi
```

Comment:
Can you please use multiline for this `if` the same way you did above?

Also what do you think about this way, to align the indention?

```Dockerfile
RUN if [ "$debug" = "true" ] ; then  \
      WEB_CONCURRENCY=0 ; \
    fi
```

Reply:
this change makes me think that what @oburstein-hub is really looking for is the dev Dockerfile [here](https://github.com/cyberark/conjur/blob/master/dev/Dockerfile.dev)